src/Features/CSharp/Portable/InitializeParameter/CSharpAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -49,5 +56,14 @@ protected override bool CanOffer(SyntaxNode body)
 
             return true;
         }
+
+        protected override SyntaxNode GetParameterNodeAtIndex(int ordinal, SyntaxNode functionDecleration)
+        {
+            var listOfParameters = InitializeParameterHelpers.GetParameters(functionDecleration);
+
+            return listOfParameters.Length < ordinal
+                ? null
+                : ordinal > listOfParameters.Length && listOfParameters[ordinal] != null ? null : listOfParameters[ordinal];
```

Comment:
nested conditional operators is hard to read, consider breaking it apart into separate statements.